### PR TITLE
Modify dataset IRIs

### PR DESCRIPTION
**Pull request by justaddcoffee**
This PR alters the dataset summary, version, and distribution IRIs from:
```
https://archive.monarchinitiative.org/[source]
https://archive.monarchinitiative.org/[source]_[version]
https://archive.monarchinitiative.org/[source]_[version].[distribution file type]
```
To what Tom suggests:
```
https://archive.monarchinitiative.org/#[source]
https://archive.monarchinitiative.org/[version]/#[source]
https://archive.monarchinitiative.org/[version]/rdf/[source].[distribution type]
```
For example for RGD these IRIs will now be: 

```
https://archive.monarchinitiative.org/#rgd
https://archive.monarchinitiative.org/201908/#rgd
https://archive.monarchinitiative.org/201908/rdf/rgd.ttl
```

n.b. Dipper doesn't consider the CURIE versions of these IRIs (e.g. `MonarchArchive:/#rgd`) to be valid CURIEs, but will write them out anyway if told to do so

### CODE REVIEW

File: dipper/models/Model.py [hunk 1]
```diff
@@ -218,7 +218,10 @@ def addOntologyDeclaration(self, ontology_id):
             ontology_id, self.globaltt['type'], self.globaltt['ontology'])
 
     def addOWLVersionIRI(self, ontology_id, version_iri):
-        self.graph.addTriple(ontology_id, self.globaltt['version_iri'], version_iri)
+        self.graph.addTriple(ontology_id,
+                             self.globaltt['version_iri'],
+                             version_iri,
+                             object_is_literal=False)
```
Comment (justaddcoffee) [inline dipper/models/Model.py]: @TomConlin interested in your opinion here. I'm having to add this to force the dataset version IRI (e.g. `MonarchArchive:/201908/#OMIM`) to be considered an IRI and not a literal. This change means that any ingest that sends a OWL version IRI that isn't a curie will fail with an `AssertionError` when Dipper goes to actually add the triple. That is okay I think, I'm just curious where you suggest we draw the line about when to make an ingest fail versus emit a warning and recover.

Comment (TomConlin) [reply dipper/models/Model.py]: I have done the same but within various helper functions that are intended to only emit non literal objects. What do you mean it must be a curie?  Seems sending a full iri should work as well as a curie at not being a literal.

Comment (justaddcoffee) [reply dipper/models/Model.py]: > What do you mean it must be a curie? Seems sending a full iri should work as well as a curie at not being a literal.

Yep, should've said CURIE or IRI

> I would make it fail if version_iri could not be a proper IRI or a curie.

Okay, good, thanks. That's the behavior here

**Comment by TomConlin** [comment]
can re open if it is decided we should do something else

**Pull request merged**